### PR TITLE
Task 3 (First API with AWS API Gateway and AWS Lambda)

### PR DESCRIPTION
**Pull request by Y-V-H**
****My account is suspended. I'm working with it**** 

**1) Was done:**
- create two functions in separate files (getProductsList & getProductsById);
- request for getting products works correct - https://j51idxfyuh.execute-api.eu-west-1.amazonaws.com/dev/products.
- FE integration works correct - http://my-first-rs-aws-app.s3-website-eu-west-1.amazonaws.com/
- getProductsById - https://j51idxfyuh.execute-api.eu-west-1.amazonaws.com/dev/product/7567ec4b-b10c-48c5-9345-fc73c48a80a2

Additional (optional) tasks

- modules are used 
- Lambda handlers (getProductsList, getProductsById) code is written not in 1 single module (file)
- Main error scenarios are handled by API ("Product not found" error) - https://j51idxfyuh.execute-api.eu-west-1.amazonaws.com/dev/product/7567ec4b-b10c-48c5-9345-fc73c48a80

FE PR - https://github.com/Y-V-H/aws-cours-fe/pull/2

JFYI: I have an issue with CloudFront which does not fixed yet
image
image

### CODE REVIEW

File: photo-store/functions/getProductsById.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+module.exports.handler = async (event) => {
+  const { id } = event.pathParameters;
+  const {products} = require('../mock/all-procuts.js');
+
+  // console.log(typeof(products), "products")
+  try{
+    let result = products.filter((el) => {
```
Comment (varchenkojulia1) [inline photo-store/functions/getProductsById.js:7]: can use .find and check if find something> result 200, if no, 400. Do we need try / catch?

File: product-service/handler.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports.hello = async (event) => {
```
Comment (varchenkojulia1) [inline product-service/handler.js:3]: do we have this in task?

Comment (Y-V-H) [reply product-service/handler.js:3]: If you mean the hello function, it was just a test function that I decided to don't delete :)

File: photo-store/functions/getProductsById.js (hunk 1 above)
Comment (Y-V-H) [reply photo-store/functions/getProductsById.js:7]: Thank for your advice and notice

**Pull request merged**